src/confluent_kafka/schema_registry/avro.py

```diff
@@ -62,8 +62,8 @@ def _schema_loads(schema_str):
     schema_str = schema_str.strip()
 
     # canonical form primitive declarations are not supported
-    if schema_str[0] != "{":
-        schema_str = '{"type":"' + schema_str + '"}'
+    if schema_str[0] != "{" and schema_str[0] != "[":
+        schema_str = '{"type":' + schema_str + '}'
```

Comment:
The code previously added quotes around schema_str, but now does not.

Reply:
Yes, I believe this was actually a bug. According to the Avro spec, schemas should be valid JSON documents, so putting quotes around the schema string duplicates double quotes for canonical form primitives. We encountered this issue when pulling schemas from our actual registry instance. This is a breaking change, but I don't believe the previous behavior was correct. If you don't want to break compatibility we could check for the first character being a double quote and conditionally add quotes based on that check?

Reply:
I just tested registering the schema `string`, with and without quotes in Confluent Schema Registry. It rejects the version without quotes (because it is not a valid avro schema) and accepts the version with quotes. I don't see how there would be any scenario in where there is working code relying on a `schama_str` for a primitive type doesn't have quotes, so this fixes a bug and doesn't introduce an incompatibility.